### PR TITLE
chap 7

### PR DESCRIPTION
**Pull request by trunghieu0211**

### CODE REVIEW

**Comment by trunghieu0211** [comment]
ready

File: config/routes.rb [hunk 1]
```diff
@@ -4,4 +4,6 @@
   get "/about", to: "static_pages#about"
   get "/contact", to: "static_pages#contact"
   get "/signup", to: "users#new"
+  post "/signup", to: "users#create"
+  resources :users, only: [:create, :show]
```
Comment (baclv) [inline config/routes.rb]: dùng `%i(create show)` nhé bạn
từ khoá google "ruby percent notation"

**Comment by baclv** [comment]
commented

**Comment by trunghieu0211** [comment]
ready

**Comment by thangtx88** [comment]
Merged

**Pull request merged**